slk/sidechain_interaction.py

```diff
@@ -193,6 +198,53 @@ def _multinode_with_callback(
             callback(mc_chain, sc_chain)
 
 
+def _external_node_with_callback(
+    params: SidechainParams,
+    callback: Callable[[Chain, Chain], None],
+    setup_user_accounts: bool = True,
+) -> None:
+    assert params.mainnet_port is not None  # TODO: type this better
+    with connect_to_external_chain(
+        # TODO: stop hardcoding this
+        url=params.mainnet_url,
+        port=params.mainnet_port,
+    ) as mc_chain:
+        if params.with_pauses:
+            input("Pausing after mainchain connected (press enter to continue)")
```

Comment:
Pauses don't really make sense when the server is external. I think we can remove the mainchain pauses.